include/miopen/miopen.h

```diff
@@ -7732,6 +7733,55 @@ MIOPEN_EXPORT miopenStatus_t miopenPReLUBackward(miopenHandle_t handle,
 // CLOSEOUT RELU DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+// GLU APIs
+/** @addtogroup GLU
+ *
+ *  @{
+ */
```

Comment:
I have changed this to activation API

Reply:
@cognaiger9 My colleague @CAHEK7 means that we already have `miopenCreateActivationDescriptor`, `miopenSetActivationDescriptor`, `miopenActivationForward`, `miopenActivationBackward` etc. We need to consider the possibility of integrating the new algorithm into existing APIs. If you have a suggestion on how this could be implemented, or, on the contrary, an objection as to why this should not be done, please feel free to express it. We'll also discuss this internally.

Reply:
The API for the GLU operation doesn't conform to current activation API as it introduces the `dim` parameter. I would see the GLU as a special case of activation so integrate it into existing APIs would break the generality of API.